### PR TITLE
Add Data Specs to the main menu, move Github to a corner

### LINKED ISSUE #114: data-semantics
Hi,

data-semantics (http://opentracing.io/data-semantics/) page is very informative, and would be great to have a link to it along with specifications (on the left menu) or indexed in the spec. It is bit hard to discover that link, but is a good resource to frequently refer to.

thanks.
bsr.

### PR DESCRIPTION
**Pull request by yurishkuro**
Fixes #114 

@bensigelman @bsr203

### CODE REVIEW

**Comment by yurishkuro** [comment]
image

image

File: _config.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ google_analytics: UA-79528055-1
 core_pages_list:       
   "Home": '/'
   "Semantic Specification": '/spec'
+  "Data Semantics": '/data-semantics'
```
Comment (bensigelman) [inline _config.yml]: How about `"Conventions and Standard Names"`? (if it fits)

We could rename the `data-semantics` file and title if you agree.

Sorry to split hairs, but "Data Semantics" is so vague...

**Comment by bensigelman** [comment]
Thanks for the github thing!

LG aside from my naming quibble.

**Comment by bensigelman** [comment]
@yurishkuro "Data Conventions" SGTM

**Comment by adriancole** [comment]
LG

**Pull request merged**